acquire/acquire.py

```diff
@@ -584,6 +584,24 @@ class Tasks(Module):
     ]
 
 
+@register_module("-ad", "--active-directory")
+class ActiveDirectory(Module):
```

Comment:
```suggestion
class ActiveDirectory(Module):
    DESC = "Active Directory data (policies etc.)"
```
Or whatever descriptive message you can come up with 😄.